src/compressed_tensors/utils/safetensors_load.py

```diff
@@ -131,31 +131,42 @@ def get_weight_mappings(model_path: str) -> Dict[str, str]:
 
     This generalizes to cases where the model is split into multiple safetensors files
 
-    :param model_path: path to safetensors state dict, must contain either a single
-    safetensors file or multiple files with an index
+    :param path_to_model_or_tensors: path to directory that contains
+        safetensors (must contain either a single file or multiple files with an index),
+        or a path to a single safetensors file
```

Comment:
Maybe I'm missing something, I don't see why we need to support passing in the safetensors file directly here? It seems like it just makes it more complicated

Reply:
On the one hand, I agree with you. The main pathway for most ML practitioners is to always read a folder with one or more weight files and load them into the model. But for general usability, it makes sense to be able to load just one tensor file, like you would in numpy or torch.